Sources/Private/CoreAnimation/Animations/GradientAnimations.swift

```diff
@@ -193,12 +193,12 @@ extension GradientShapeItem {
       for colorIndex in 0..<numberOfColors {
         let colorStartIndex = colorIndex * 4
 
-        let colorLocation = CGFloat(colorComponents[colorStartIndex])
+        let colorLocation = Double(colorComponents[colorStartIndex])
```

Comment:
banished `CGFloat` to the shadow realm

Reply:
One of these `CGFloat` changes causes the Xcode 13.2 build to fail with a weird and unhelpful `Illegal instruction: 4` error, so I reverted those changes